neptune/experiments.py

```diff
@@ -626,6 +638,21 @@ def get_properties(self):
         return dict((p.key, p.value) for p in experiment.properties)
 
     def set_property(self, key, value):
+        """Sets a value of an experiment property. If property with given key does not exist it adds a new one.
```

Comment:
put this in new line: If property with given key does not exist it adds a new one.